doc/nrf/gs_modifying.rst

```diff
@@ -150,9 +231,30 @@ See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how
 To locate a specific configuration option, use the filter (:guilabel:`Jump to` in menuconfig and guiconfig).
 The documentation for each :ref:`configuration option <configuration_options>` also lists the menu path where the option can be found.
 
-.. important::
-   All changes to the :file:`.config` file are lost when you clean your build directory.
-   You can save it to another location, but you must then manually copy it back to your build directory.
+Changing the configuration permanently
+======================================
+
+To configure your application and maintain the configuration when you clean the build directory pristinely, you need to specify the configuration in one of the permanent configuration files.
+In most cases, this will be either the default :file:`prj.conf` file of the application or an extra Kconfig fragment.
+In these files, you can specify different values for configuration options that are defined by a library or board, and you can add configuration options that are specific to your application.
+
+See :ref:`zephyr:setting_configuration_values` in the Zephyr documentation for information on how to change the configuration permanently.
+
+.. tip::
+   Reconfiguring through menuconfig only changes the specific setting and will not adjust defaults for other visible and configured symbols.
+   This may result in a bloated configuration compared to changing the setting directly in :file:`prj.conf`.
+   See the section Stuck symbols in menuconfig and guiconfig on the :ref:`kconfig_tips_and_tricks` in the Zephyr documentation for more information.
+
+If you work with SES, the :file:`prj.conf` file is read when you open a project.
+This means that after you edit this file, replace it with the newer version by using the :guilabel:`Project` > :guilabel:`Run CMake...` option.
```

Comment:
This is a bit imprecise. The file will be read when CMake re-runs, which it will do automatically when a re-build is requested by the user and `prj.conf` has changed, but it can also be done manually if users want the settings updated earlier.

```suggestion
The file will be reloaded when CMake re-runs. This will happen automatically when the application is rebuilt, but can also be requested manually by using the :guilabel:`Project` > :guilabel:`Run CMake...` option.
```